docs/how-to/ocm-control-plane-walkthrough.md

```diff
@@ -136,18 +134,29 @@ Open three windows, which we'll refer to throughout this walkthrough as:
 
 ### Start the Gateway Controller
 
-1. In `T1` run the following to start the Gateway Controller:
+1. In `T1` run the following to build and deploy the Gateway Controller in a container:
 
     ```bash
     make build-controller kind-load-controller deploy-controller
     ```
 
+    *Alternatively, in a new window we'll call `T3`, you can run the controller locally instead by running:*
+
+    ```bash
+    (export $(cat ./controller-config.env | xargs) && export $(cat ./aws-credentials.env | xargs) && make build-controller install run-controller)
+    ```
+
 ### Check the managed zone
 
 1. Lets ensure our `managedzone` is present. In `T1`, run the following:
 
     ```bash
     export KUBECONFIG=$(pwd)/local/kube/control-plane.yaml
```

Comment:
When I ran this command it actually knocked of my KUBECONFIG. I hadn't run the command on line 67 because it said optional. i was able to run other `kubectl` commands in the meantime.